schema/properties_directory/non-scalar/hubbard_u.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$id": "properties-directory/non-scalar/hubbard-u",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Hubbard U parameters",
+    "description": "Hubbard U value in eV",
+    "properties": {
+        "name": {
+            "enum": ["hubbard_u"]
+        },
+        "units": {
+            "enum": ["eV"]
+        },
+        "values": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "allOf": [
+                    {
+                        "$ref": "../../core/reusable/atomic_data_per_orbital_numeric.json"
+                    }
+                ],
+                "required": ["id", "atomicSpecies", "orbitalName", "value"]
```

Comment:
Nit: I think we previously had a requirement to have all array items on newlines, not a huge deal, but would be good to be consistent. We can also put a task to add a JSON linter.